plugins/zenoh-backend-traits/src/config.rs

```diff
@@ -45,10 +46,55 @@ pub struct StorageConfig {
     pub strip_prefix: Option<OwnedKeyExpr>,
     pub volume_id: String,
     pub volume_cfg: Value,
+    // Note: ReplicaConfig is optional. Alignment will be performed only if it is a replica
+    pub replica_config: Option<ReplicaConfig>,
     // #[as_ref]
     // #[as_mut]
     // pub rest: Map<String, Value>,
 }
+// Note: All parameters should be same for replicas, else will result on huge overhead
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ReplicaConfig {
+    pub align_prefix: String,
+    pub publication_interval: Duration,
+    pub propagation_delay: Duration,
+    pub delta: Duration,
+    pub subintervals: usize,
+    pub hot: usize,
+    pub warm: usize,
+}
+
+impl Default for ReplicaConfig {
+    fn default() -> Self {
+        Self {
+            // This is the prefix that is used by the alignment protocol
+            // TODO: discuss if this needs to be configurable
+            align_prefix: String::from("@-digest"),
```

Comment:
Removed `align_prefix` option from config. Instead, it is built-in to the system as it is a system-wide setting. Easy for users to mess with otherwise.